### PR TITLE
Add backend error and constraints translations via new header

### PR DESCRIPTION
**Pull request by aerrasti**
## Subject
Some backend api error and constraints can not be translated. That's why we are using the `Accept-Language` to set locale in api requests.

### CODE REVIEW

**Review by eddsaura — DISMISSED** [review]

File: frontend/contexts/StooaManager.tsx [hunk 1]
```diff
@@ -44,6 +44,7 @@ const StooaProvider = ({ data, isModerator, children }) => {
   const [conferenceReady, setConferenceReady] = useState(false);
   const { addToast, clearDelayed } = useToasts();
   const { t, lang } = useTranslation('app');
+  const { locale } = useRouter();
```
Comment (eddsaura) [inline frontend/contexts/StooaManager.tsx:47]:
```suggestion

```

File: frontend/contexts/StooaManager.tsx [hunk 2]
```diff
@@ -83,7 +84,9 @@ const StooaProvider = ({ data, isModerator, children }) => {
 
   const checkApIConferenceStatus = () => {
     api
-      .get(`${lang}/fishbowl-status/${fid}`)
+      .get(`${lang}/fishbowl-status/${fid}`, {
+        headers: { 'Accept-Language': locale }
```
Comment (eddsaura) [inline frontend/contexts/StooaManager.tsx:88]:
```suggestion
        headers: { 'Accept-Language': lang }
```

File: frontend/hocs/withIsFishbowlEnded.js [hunk 3]
```diff
@@ -25,10 +25,13 @@ const withIsFishbowlEnded = WrappedComponent => props => {
   const router = useRouter();
   const { lang } = useTranslation();
   const { fid } = router.query;
+  const { locale } = useRouter();
```
Comment (eddsaura) [inline frontend/hocs/withIsFishbowlEnded.js:28]:
```suggestion

```

File: frontend/hocs/withIsFishbowlEnded.js [hunk 4]
```diff
@@ -25,10 +25,13 @@ const withIsFishbowlEnded = WrappedComponent => props => {
...
 
   useEffect(() => {
     api
-      .get(`${lang}/fishbowl-status/${fid}`)
+      .get(`${lang}/fishbowl-status/${fid}`, {
+        headers: { 'Accept-Language': locale }
```
Comment (eddsaura) [inline frontend/hocs/withIsFishbowlEnded.js:33]:
```suggestion
        headers: { 'Accept-Language': lang }
```

**Review by eddsaura — COMMENTED** [review]
Good job Aitor 👍 

To finnish pollishing this I think we should use lang from next-translate where we already have it. Check it and test if it works this way properly.

**Review by eddsaura — DISMISSED** [review]

**Comment by aerrasti** [comment]
> Good job Aitor 👍
> 
> To finnish pollishing this I think we should use lang from next-translate where we already have it. Check it and test if it works this way properly.

done :)

**Review by eddsaura — APPROVED** [review]

**Pull request merged**